docs/_includes/text-mono.adoc

```diff
@@ -112,7 +112,7 @@ While the previous examples will work in Asciidoctor 1.5.0, the last two are now
 To use the transitional syntax, add the role "x-" in front of the legacy syntax to indicate that you want Asciidoctor 1.5.0 to use the old behavior. Of course, Asciidoctor 0.1.4 already understands the old syntax.
 
 * [x-]\`Use {asciidoctor-version} to print the version of Asciidoctor` (attribute is _not_ replaced)
-* [x-]+The version of Asciidoctor is {asciidoctor-version}+ (attribute _is_ replaced)
+* [x-]`The version of Asciidoctor is {asciidoctor-version}` (attribute _is_ replaced)
```

Comment:
This one we don't want to replace because it is showing how to use the backwards-compatible syntax.